crates/oxc_language_server/src/capabilities.rs

```diff
@@ -51,6 +66,10 @@ impl From<Capabilities> for ServerCapabilities {
             } else {
                 None
             },
+            execute_command_provider: Some(ExecuteCommandOptions {
```

Comment:
Only provide the Commands when the server supports them (reflecting the client support)